client/component-lab/TrialChargeInfo/TrialChargeInfo.coffee

```diff
@@ -19,7 +19,7 @@ TrialChargeInfo = ({ teamSize, pricePerSeat, daysLeft }) ->
           Team Size: <strong>{pluralize 'Developer', teamSize, yes}</strong>
         </Label>
       </Col>
-      <Col xs={7} className={textStyles.right}>
+      <Col xs={7} className={textStyles.right + ' hidden'}>
```

Comment:
didn't want to break anything, these were blind changes I couldn't even test since I didn't have a post trial team locally.